lib/Jsrt/Jsrt.cpp

```diff
@@ -2799,9 +2799,12 @@ CHAKRA_API JsCreateNamedFunction(_In_ JsValueRef name, _In_ JsNativeFunction nat
     });
 }
 
-void SetErrorMessage(Js::ScriptContext *scriptContext, JsValueRef newError, JsValueRef message)
+void SetErrorMessage(Js::ScriptContext *scriptContext, Js::JavascriptError *newError, JsValueRef message)
 {
-    Js::JavascriptOperators::OP_SetProperty(newError, Js::PropertyIds::message, message, scriptContext);
+    Js::PropertyDescriptor desc;
+    desc.SetValue(message);
```

Comment:
For ease-of-viewing, I'd appreciate if this also explicitly set configurable and writable to true, to match the spec.